interface/batchcom/settingsnotification.php

```diff
@@ -1,31 +1,54 @@
 <?php
+/**
```

Comment:
For this file, mimick the changes you made regarding license/author header, html escaping, sql-injection, and translations that you made above in interface/batchcom/emailnotification.php